packages/utilities/psammead-storybook-helpers/README.md

```diff
@@ -72,6 +76,28 @@ storiesOf('Caption', module)
   );
 ```
 
+### dirDecorator
+
+```jsx
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+import { withKnobs } from '@storybook/addon-knobs';
+import { dirDecorator } from '@bbc/psammead-storybook-helpers';
+import Brand from '@bbc/psammead-brand';
+import { news as brandSVG } from '@bbc/psammead-assets';
+
+storiesOf('Brand', module)
+  .addDecorator(withKnobs)
+  .addDecorator(dirDecorator)
+  .add(
+    'default',
+    (inputValues, script, dir) => (
```

Comment:
This example doesnt need `(inputValues, script, dir) => (`

Can you make this a really simple example please, just with `<h1>Lorem Ipsum</h1>` please